src/languages/html.coffee

```diff
@@ -79,7 +79,27 @@ module.exports = {
         ]
       items:
         type: 'string'
-      description: "List of tags (defaults to inline) that should not be reformatted"
+      description: "(Depracated for most scenarios) List of tags that should not be reformatted at all.  NOTE: Set this to [] to get improved beautifier behavior."
```

Comment:
Typo: `Deprecated`

Reply:
Re: Note about improved behaviour.
Do you want to mention `inline` option at all?